src/elements/Table/TableCell.tsx

```diff
@@ -1,32 +1,18 @@
-'use client';
-
 import type { TableCellNode } from '@prezly/story-content-format';
 import classNames from 'classnames';
 import type { HTMLAttributes } from 'react';
-import { useMemo } from 'react';
-
-import { useTableContext } from './TableContext';
 
 interface Props extends HTMLAttributes<HTMLTableCellElement> {
-    node: TableCellNode;
+    node: TableCellNode & { isHeader?: boolean };
```

Comment:
Good point, had to update the match function as well though because otherwise it was complaining that `isHeader` does not exist on `TableCellNode`: 94f66948107b3ecafc626e08d0232165fdfafa9d